contribs/gmf/apps/desktop_alt/index.html.ejs

```diff
@@ -404,7 +424,7 @@
       </div>
     </footer>
     <script src="https://cdn.polyfill.io/v2/polyfill.min.js?features=es6,default-3.6,Array.prototype.includes,Object.entries,Object.values" crossorigin="anonymous"></script>
-    <script src="https://maps.googleapis.com/maps/api/js?v=3&key=AIzaSyA3NVIy-HOYT0a0CkChA6nFwqEFqHYWBVk" crossorigin="anonymous"></script>
```

Comment:
@sbrunner Do you think, removing crossorigin="anonymous" could create any problem?

Reply:
No, it shouldn't :-)